tsstats.sample.ini

```diff
@@ -33,7 +33,10 @@ owners: uid1
 commandprefix: !
 
 ;uid of clients you don't want to be counted, separate multiple uids by commas (no spaces)
-ignore : XXXXXXXXXXXXXXXXXXX
+igclient: XXXXXXXXXXXXXXXXXXX
```

Comment:
I'd prefer not to change existing options, to minimize the risk of existing configurations not working anymore because of a rename